### PR TITLE
Create package adapter

### LINKED ISSUE #11: Add Adapters to handle request data
The idea is to create functions to receive the request data, then validate and return the data on the right format to be used on the use case.

### PR DESCRIPTION
**Pull request by LucasEduardoGiovanini**
I forgot to create the pull request as soon as I created the package, so i'm creating now. I'm alredy concluded the creation of adapter package and refactoring of views functions who now need of adapter.

I divided it into 3 parts:
-create the adapter package  (done)
-remove the auth classes (done)
-refactore the function token_required to return the email extracted from token (done)

closes #11

### CODE REVIEW

**Comment by LucasEduardoGiovanini** [comment]
Good morning johnny, my code is ready for analysis.

File: Desafio_Snowman/repositories/ponto_turistico_repository.py [hunk 1]
```diff
@@ -30,8 +30,10 @@ def search_points(self):
     def check_existence_of_the_point(self, name: str):
         cursor = self.connection.cursor()
         arguments = (name,)
+
         cursor.execute("SELECT nome, categoria, latitude, longitude FROM tbPontoTuristico WHERE nome = %s", arguments)
         resultado = cursor.fetchone()
+        print(resultado)
```
[the remark this answers is not in the document]
Comment (LucasEduardoGiovanini) [reply Desafio_Snowman/repositories/ponto_turistico_repository.py:36]: Sorry for this, i was testing if result received some value and forgot to remove this print.

File: Desafio_Snowman/adapter.py [hunk 2]
```diff
@@ -1,34 +1,30 @@
 from flask import request
```
Comment (johnnywell) [inline Desafio_Snowman/adapter.py:1]: You don't need it anymore.

**Comment by LucasEduardoGiovanini** [comment]
I need to get used to removing imports that I no longer use

**Pull request merged**